test/test_sparse_csr.py

```diff
@@ -955,6 +957,23 @@ def test_coo_csr_conversion(self, device, dtype):
 
             self.assertEqual(csr_sparse.to_dense(), dense)
 
+    @ops(_sparse_csr_ops)
+    def test_sparse_csr_no_error(self, device, dtype, op):
+        samples = op.sample_inputs(device, dtype)
+
+        if len(samples) == 0:
+            self.skipTest("Skipped! No sample inputs!")
+
+        for sample in samples:
+            assert isinstance(sample.input, torch.Tensor)
+            if sample.input.ndim != 2:
```

Comment:
Would it be better to filter the inputs in `sample_input_fn` with `op_info.supports_sparse_csr` (as suggested by @IvanYashchuk ) and raise an error here. (This might lead to silent success as mentioned by @krshrimali in the meeting)